deploy/dotcom/prod/Dockerfile

```diff
@@ -37,8 +37,14 @@ COPY --from=elixir-builder /root/deps /root/deps
 ADD ./priv/static /root/priv/static
 ADD ./assets /root/assets
 
+RUN npm i -g husky@latest
+
 WORKDIR /root/assets
+
+RUN npm install --package-lock-only
 RUN npm ci --ignore-scripts
+RUN npm install -S -install-links ../deps/mbta_metro/priv/
+
```

Comment:
📓 This just does all three of those things in the build process.

Reply:
cool. we should probably create an alias to group these commands at some point